app/models/user.rb

```diff
@@ -210,6 +210,12 @@ def self.authenticate(login, password)
     end
   end
 
+  def self.custom_fields
+    fields = FoodsoftConfig[:custom_fields] && FoodsoftConfig[:custom_fields][:user]
+    return [] unless fields
+    fields.map(&:deep_symbolize_keys)
+  end
+
```

Comment:
This looks like it could benefit if becoming a concern?

Reply:
i rebased it to the current `master`